### ISSUE TITLE
What markup how?

### ISSUE THREAD

**Issue by simonmika** [cogneco/typeup#4]
I would love some input regarding how what features to build first and especially get some feedback and ideas on **how to mark them up**.

**Comment by simonmika** [comment]
To get a general overview please read the [readme](https://github.com/cogneco/typeup/blob/master/README.md).

To see how it looks like today look at the [example](https://github.com/cogneco/typeup/blob/master/examples/document.tup).

And for some of the bigger features planned there are already some issues:
- [Math](./1) 
- [Code](./2)
- [Captions](./3)

But we also need ways to markup things like links, figures, tables and more.

**Comment by simonmika** [comment]
@fredrikbryntesson @bettinaselig @thomasfanell @davidhesselbom @jonathanudd @ericleimar @christopherlagerhult @marcusnaslund @johannesl @sannahmika @sikabo @ishaimika

Any ideas or comments?
Anybody else you who's opinion you think would be valuable?

**Comment by davidhesselbom** [comment]
I don't see numbered lists being mentioned anywhere. Will items be numbered manually with `1.`, `2.` etc, or will you just put `1.` everywhere and let the items be numbered automatically? Markdown allows you to choose, but if TypeUp only allows for one way of doing things, you'll have to decide.

**Comment by davidhesselbom** [comment]
Also, I don't remember where, but I've seen Markdown refuse to render lists properly unless subitems are indented by 2 spaces. I see you're using tabs in your example. I'm not sure which is better.

**Comment by simonmika** [comment]
I think allowing any number followed by a dot and a space makes sense. The order is then automatically generated at presentation.

**Comment by simonmika** [comment]
I would like to use space and/or tabs to make nested lists and also handle multiple paragraph in a list item. What do you think, spaces or tabs?

**Comment by davidhesselbom** [comment]
> and also handle multiple paragraph in a list item.

Yeah, this is missing in Markdown, so you end up having to number list items manually whenever you want to do this.

> I think allowing any number followed by a dot and a space makes sense. The order is then automatically generated at presentation.

You mean the manually entered numbers will be ignored, then? Works for me.

> What do you think, spaces or tabs?

Not sure. Spaces, probably.
